pyscript.core/tests/integration/test_02_display.py

```diff
@@ -202,8 +202,8 @@ def test_multiple_display_calls_same_tag(self):
             """
             <script type="py">
                 from pyscript import display
-                display('hello')
-                display('world')
+                display('hello', append=True)
```

Comment:
same here ... if we really want to be sure `append=True` is the default, we should keep this not touched to me